sdk/core/azure-core/test/e2e/CMakeLists.txt

```diff
@@ -1,48 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# SPDX-License-Identifier: MIT
-
-if (BUILD_TRANSPORT_CURL)
-
-cmake_minimum_required (VERSION 3.13)
-set(TARGET_NAME "azure_core_with_curl")
-set(TARGET_NAME_STREAM "azure_core_with_curl_stream")
-set(TARGET_NAME_STORAGE_ISSUE_249 "azure_core_storage_issue_249")
-set(TARGET_NAME_STORAGE_ISSUE_248 "azure_core_storage_issue_248")
```

Comment:
Hopefully these issue fixes are covered with regression tests.

Reply:
Nop, we need to add this as Live tests on Storage instead.